rhizome/host/bin/prep_host.rb

```diff
@@ -4,6 +4,17 @@
 require_relative "../../common/lib/util"
 require_relative "../lib/cloud_hypervisor"
 require "fileutils"
+require "socket"
+
+original_hostname = Socket.gethostname
```

Comment:
nit: we can do this after ensuring the hostname is passed (i.e. after the check below)